Source/WebKit/Scripts/generate-derived-log-sources.py

```diff
@@ -93,11 +93,11 @@ def generate_message_receiver_implementations_file(log_messages, log_messages_re
             if category == "Default":
                 file.write("    auto osLogPointer = OS_LOG_DEFAULT;\n")
             else:
-                file.write("    auto osLog = adoptOSObject(os_log_create(\"com.apple.WebKit.WebContent\", \"" + category + "\"));\n")
+                file.write("    String subsystem = makeString(\"com.apple.WebKit.WebContent.\"_s, String::number(m_pid));\n")
```

Comment:
This is unnecessarily inefficient, you don't need `String::number()`, just include `<wtf/text/StringConcatenateNumbers.h>`